npm-upgrade.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
and again .. what even is ncu ?

Reply:
npm install -g npm-check-updates = ncu to get automated upgrades to all dependencies of the project - I could set it to be ignored in the .gitignore - helpfully in development to check and/or update dependencies